build/tasks/TrimDeps.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Linq;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace RepoTasks
+{
+    public class TrimDeps : Task
```

Comment:
What are we trimming and why? Is this a workaround for limitations in dotnet-store?

Reply:
This trims the first entry which is the entry assembly, i.e. `HostingStartup` which is the template used to generate the deps file and we don't want to include it.